Development

MoxElliot opened the pull request.

valanex committed:
Add shapes assets for shapes game

MoxElliot committed:
start of day after pull development

MoxElliot committed:
Failure Btn Styling

Failure btn styling now works, red and strike through. But score isn't calculating correctly as trynum seems to be broken

MoxElliot committed:
Removed tryNum, id

To fix score but, replaced tryNum conditional with failure.length conditional which pulls from the slice state. to Fix (I think) the 'empty string' bug with e.target.id I replaced it with answerArr[btnid]

E-wang42 committed:
update

E-wang42 committed:
Merge branch 'development' of https://github.com/chingu-voyages/v42-geckos-team-22 into ui_consistency

E-wang42 committed:
update

E-wang42 committed:
update heading text

E-wang42 committed:
update favicon

E-wang42 committed:
update: mobile menu

Danneytrieu committed:
sign in

Danneytrieu committed:
update animation

Danneytrieu committed:
clean up code

E-wang42 committed:
add to slice folder

MoxElliot committed:
merge conflict resolution

MoxElliot committed:
added number pixabay apis in data.js

E-wang42 committed:
update logo

MoxElliot committed:
data.js number, animal, letter

added data for images, correct answer, and choices for numbers, animals, and letters

MoxElliot committed:
NavBar update

Updated navbar (dropdown.jsx, Header.jsx) to link to each game type

MoxElliot committed:
added API to GamePlayWindow

Renamed api component to GetApiPics, imported into GamePlayWindow in place of current <img/>

E-wang42 committed:
fix: success modal ui

MoxElliot committed:
End of day

MoxElliot committed:
API works and Image loads

Needed to add dependencies in the api.jsx useeffect dependency array for image, currentQuestionImage

MoxElliot committed:
Set Image Size

max width and height set to 320px, image tag set to w-fit h-full p-8

MoxElliot committed:
added shapes

MoxElliot committed:
Merge pull request #41 from chingu-voyages/feature_signin

Feature signin with firebase

MoxElliot committed:
merge resolved

E-wang42 committed:
update

Danneytrieu committed:
Merge pull request #38 from chingu-voyages/feature_game-mode-styling

Feature game mode styling

Danneytrieu committed:
Merge pull request #32 from chingu-voyages/assets/shapes

Add shapes assets for shapes game

MoxElliot committed:
Adding back landing/loading window

MoxElliot committed:
merge conflict resolved

MoxElliot committed:
merge conflict resolved

MoxElliot committed:
Merge pull request #42 from chingu-voyages/feature_game-images-api

Feature game images api

MoxElliot committed:
Loading, Landing, Links

Loading and Landing screen are back. Clicking drop down game links resets game state (to landing), score, and question

MoxElliot committed:
Play Button now Displays Game Name

MoxElliot committed:
Button now displays correct game type

MoxElliot committed:
merge conflict resolved

valanex committed:
First draft of FAQ content

E-wang42 committed:
update

E-wang42 committed:
update

E-wang42 committed:
everything still works

Danneytrieu committed:
Merge pull request #46 from chingu-voyages/ui_consistency

UI consistency

Danneytrieu committed:
Merge branch 'development' into feature_game-loading-screen

Danneytrieu committed:
Merge pull request #44 from chingu-voyages/feature_game-loading-screen

Feature game loading screen

Danneytrieu committed:
prep for deployment

Danneytrieu commented:
-fixed borders to match design 
-added signin/signout/register to nav mobile
-added display name to game header
-fix faq svg not showing correctly 
-fix import headerslice issue
-fix animal homepage routing

MoxElliot committed:
pulled to update

valanex committed:
Finalize FAQ content

MoxElliot committed:
added/removed/fixed shapes

Corrected oval import, added square, triangle, circle, removed rectangle

MoxElliot committed:
correct answer button disabled

correct answer button now disables onClick via setState, resets to enabled after setTimeout runs via setState again

MoxElliot committed:
refactored score screen

total score now displays as 'Your Score is: x/y points!'

MoxElliot committed:
created GameSpinnerWindow

MoxElliot committed:
added comments in api.jsx to figure this out

MoxElliot committed:
Merge branch 'fix_game-issues' of https://github.com/chingu-voyages/v42-geckos-team-22 into feature_enable-loading-spinner

MoxElliot committed:
Adjusted async in api.jsx to add loading state

MoxElliot committed:
loading window works

Loading screen shows between flashcards, needs centering

MoxElliot committed:
loading screen works

E-wang42 committed:
add loading spinner function

E-wang42 committed:
Merge branch 'development' into api_loading

E-wang42 committed:
fix: add borders

valanex committed:
Remove height spec for accordion to push page down

valanex committed:
Fix gap under footer

valanex committed:
Merge branch 'development' into feature/faq-content

E-wang42 committed:
Merge pull request #47 from chingu-voyages/feature/faq-content

Feature/faq content

valanex committed:
Merge pull request #49 from chingu-voyages/api_loading

Loading Spinner Api Request

Danneytrieu committed:
Merge pull request #50 from chingu-voyages/border-fix

Border fix

E-wang42 committed:
fix: no ring on md

E-wang42 committed:
Merge branch 'development' of https://github.com/chingu-voyages/v42-geckos-team-22 into text-nowrap

E-wang42 committed:
fix: text no-wrap

MoxElliot committed:
Merge branch 'development' of https://github.com/chingu-voyages/v42-geckos-team-22 into development

MoxElliot committed:
merge conflict resolved

MoxElliot committed:
Home Page to game Fix

Added the handleToggleGames, useDisptach, useSelector suite from DropDown.jsx to HomePage links. Now the game displays loading/landing page as expected when user clicks home page links

MoxElliot committed:
Error fix

removed empty dependency array in GameLoadingWindow useEffect. Fixed bad set state call error in console

MoxElliot committed:
Merge pull request #51 from chingu-voyages/text-nowrap

Text nowrap

MoxElliot committed:
Merge branch 'development' of https://github.com/chingu-voyages/v42-geckos-team-22 into development

MoxElliot committed:
removed console.logs, unneccessary comments, unused imports

MoxElliot committed:
Removed toggleMenu from home screen links

E-wang42 committed:
Merge pull request #48 from chingu-voyages/fix_game-issues

Fix game issues

MoxElliot committed:
merge conflicts

MoxElliot committed:
removed more console.log

E-wang42 committed:
Merge pull request #52 from chingu-voyages/fix_clean-up-code

Fix clean up code

E-wang42 reviewed (approved):
Only shipping bangers.
